Explorer/Assets/Scripts/ECS/Unity/Materials/Components/MaterialData.cs

```diff
@@ -0,0 +1,95 @@
+﻿using ECS.Unity.Textures.Components;
+using UnityEngine;
+
+namespace ECS.Unity.Materials.Components
+{
+    public readonly struct MaterialData
+    {
+        public readonly bool IsPbrMaterial;
+        public readonly TextureComponent? AlbedoTexture;
+        public readonly float AlphaTest;
+
+        // Cast shadows is not a part of Material, it's a part of Renderer
+        public readonly bool CastShadows;
```

Comment:
Is this a fallout of the existing structure?

Reply:
it's a part of Protocol